src/player.cpp

```diff
@@ -6142,6 +6142,65 @@ void player::suffer()
     if (has_artifact_with(AEP_ATTENTION)) {
         add_effect( effect_attention, 3 );
     }
+
+    // Stim +250 kills
+    if ( stim > 210 ) {
+        if ( one_in( 20 ) && !has_effect( effect_downed ) ) {
+            add_msg_if_player(m_bad, _("Your muscles spasm, and you fall to the ground!"));
+            add_effect( effect_downed, rng(6, 20) );
+        }
+    }
+    if ( stim > 170 ) {
+        if ( !has_effect( effect_winded ) && calendar::once_every( MINUTES(5) ) ) {
+            add_msg(m_bad, _("You feel short of breath.") );
+            add_effect( effect_winded, MINUTES(5) + 1 );
+        }
+    }
+    if ( stim > 110 ) {
+        if ( !has_effect( effect_shakes ) && calendar::once_every( MINUTES(5) ) ) {
+            add_msg( _("You start to shake uncontrollably.") );
```

Comment:
This message will reappear every 5 minutes as long as character's `stim` > 110, right? Maybe then remove the "start" part from it and leave simple "You shake uncontrollably."?
Also, I think that message reappearing every 5 minutes will easily spam the entire message log.

Reply:
Yeah, I was concerned about message spam to some extent. How far apart should the messages be spaced, then? Is 10 minutes sufficient?

Reply:
Yeah, at least 10 minutes is good imo.